README.md

```diff
@@ -24,7 +24,7 @@ In order to run this Web App locally you will need to install:
 
 Key                | Description
 -------------------|------------------------------------
-`PENALTY_PAYMENT_WEB_PORT` |The port of the Penalty-Payment-WEB service
+`PPS_PAY_WEB_PORT` |The port of the Penalty-Payment-WEB service
```

Comment:
Spot on. I will change this

> Prefer this to be kept as `PENALTY_PAYMENT_WEB_PORT` just because that is more in keep with the new repo name now. But realise there is an attempt for consistency with the -api stuff we have.
> 
> If we do want to change this - then it looks like there is a reference here in master
> 
> https://github.com/companieshouse/penalty-payment-web/blob/7d4bbb003a4d0ac490000a979722212f3266e702/start.sh#L12
> 
> See https://github.com/search?q=repo%3Acompanieshouse%2Fpenalty-payment-web%20_WEB_PORT&type=code

Spot on. I will change this